lib/services.template.ejs

```diff
@@ -37,8 +37,6 @@ if (typeof module !== 'undefined' && typeof exports !== 'undefined' &&
 
 <% for (var modelName in models) {
      var meta = models[modelName];
-     // capitalize the model name
-     modelName = modelName[0].toUpperCase() + modelName.slice(1);
```

Comment:
`modelName` should be available inside the for loop, as this deleted code just updates the variable, rather than creating a new one (the formatting that used to be in the template is now in `getFormattedModelName` in `services.js`).  I double-checked that there isn't anywhere outside the for loop where modelName is used - it's just inside this for loop that we reference it directly.